docs/doc.md

```diff
@@ -49,8 +49,22 @@
     2. If repo is re-analyzed and analyzations are up to date it has to read analyzations from the database.
 3. Has to implement resonable test.
 
+## Week 3
+1. Implement one or more integration tests in your test suite. 
+2. Illustrate the architecture of your REST service using a suitable notation.
+
 # Non Functional Requirements
 ## Week 1
 None
 ## Week 2
```

Comment:
Perhaps we should change the requirements for earlier weeks as well (since we know they're non-functional now) while we're at it